### ISSUE TITLE
Travis CI is failing with Kubernetes 1.16

### ISSUE THREAD

**Issue by roycaihw** [kubernetes-client/python#963]
e.g. 
- https://github.com/kubernetes-client/python-base/pull/163
- https://travis-ci.org/kubernetes-client/python-base/jobs/588553023

1. dynamic client is failing with the following error, we should specify the version (`v1beta1`) in the test
```
ResourceNotUniqueError: Multiple matches found for {'kind': 'CustomResourceDefinition'}: [<Resource(apiextensions.k8s.io/v1beta1/customresourcedefinitions)>, <Resource(apiextensions.k8s.io/v1/customresourcedefinitions)>]
```
2. utils test is failing creating apps/extensions APIs with the following error, but I haven't seen upstream API change that could cause this
```
FailToCreateError: Error from server (Not Found): {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"the server could not find the requested resource","reason":"NotFound","details":{},"code":404}
```

**Comment by micw523** [comment]
It’s looking like apps/v1beta1 is also gone in k8s v1.16. The CI failure first appeared when the exit code test was added and the master branch build failed, since minikube got updated to v1.4 which corresponds to k8s v1.16.

There are a bunch of things to be fixed in the tests - I’ll try churn them out. I may have to fix the k8s version to v1.15 when I prepare the patches and change the version back, since the automatically generated client still contains extensions/v1beta1 and apps/v1beta1.

**Comment by roycaihw** [comment]
> apps/v1beta1 is also gone in k8s v1.16

could you link to the upstream change? I checked the openapi spec and didn't see that
